### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by holgerd77**
Adresses #26

### CODE REVIEW

**Comment by holgerd77** [comment]
@axic what is the main problem with the ``fromV1()`` testing?

**Comment by axic** [comment]
> what is the main problem with the fromV1() testing?

Real world samples are hard to find.

**Comment by holgerd77** [comment]
@axic I'll leave this here for now, please review and don't close the issue, since this can be done incrementally. For me it was a good occasion, to get to know the library a bit better, but other things are more pressing right now.

**Comment by holgerd77** [comment]
@axic Can we have this merged?

File: test/index.js [hunk 1]
```diff
@@ -4,12 +4,19 @@ var Wallet = require('../')
...
 var ethUtil = require('ethereumjs-util')
 
-var fixturekey = Buffer.from('efca4cdd31923b50f4214af5d2ae10e7ac45a5019e9431cc195482d707485378', 'hex')
-var fixturewallet = Wallet.fromPrivateKey(fixturekey)
+var fixturePrivKey = 'efca4cdd31923b50f4214af5d2ae10e7ac45a5019e9431cc195482d707485378'
+var fixturePrivKeyStr = '0x' + fixturePrivKey
+var fixturePrivKeyBuffer = Buffer.from(fixturePrivKey, 'hex')
+
+var fixturePubKey = '5d4392f450262b276652c1fc037606abac500f3160830ce9df53aa70d95ce7cfb8b06010b2f3691c78c65c21eb4cf3dfdbfc0745d89b664ee10435bb3a0f906c'
+var fixturePubKeyStr = '0x' + fixturePubKey
```
Comment (axic) [inline test/index.js]: If `Buffer` isn't shortened, I'd have liked `PublicKey`, `PublicKeyString` and `PublicKeyBuffer` and same for private.

File: test/index.js [hunk 2]
```diff
@@ -132,14 +139,14 @@ describe('.toV3()', function () {
   var uuid = Buffer.from('7e59dc028d42d09db29aa8a0f862cc81', 'hex')
 
   it('should work with PBKDF2', function () {
-    var key = Buffer.from('efca4cdd31923b50f4214af5d2ae10e7ac45a5019e9431cc195482d707485378', 'hex')
+    var key = Buffer.from(fixturePrivKey, 'hex')
     var wallet = Wallet.fromPrivateKey(key)
```
Comment (axic) [inline test/index.js]: Could just use `fixtureWallet` here.

File: test/index.js [hunk 3]
```diff
@@ -132,14 +139,14 @@ describe('.toV3()', function () {
...
     var w = '{"version":3,"id":"7e59dc02-8d42-409d-b29a-a8a0f862cc81","address":"b14ab53e38da1c172f877dbc6d65e4a1b0474c3c","crypto":{"ciphertext":"01ee7f1a3c8d187ea244c92eea9e332ab0bb2b4c902d89bdd71f80dc384da1be","cipherparams":{"iv":"cecacd85e9cb89788b5aab2f93361233"},"cipher":"aes-128-ctr","kdf":"pbkdf2","kdfparams":{"dklen":32,"salt":"dc9e4a98886738bd8aae134a1f89aaa5a502c3fbd10e336136d4d5fe47448ad6","c":262144,"prf":"hmac-sha256"},"mac":"0c02cd0badfebd5e783e0cf41448f84086a96365fc3456716c33641a86ebc7cc"}}'
     // FIXME: just test for ciphertext and mac?
     assert.equal(wallet.toV3String('testtest', { kdf: 'pbkdf2', uuid: uuid, salt: salt, iv: iv }), w)
   })
   it('should work with Scrypt', function () {
-    var key = Buffer.from('efca4cdd31923b50f4214af5d2ae10e7ac45a5019e9431cc195482d707485378', 'hex')
+    var key = Buffer.from(fixturePrivKey, 'hex')
     var wallet = Wallet.fromPrivateKey(key)
```
Comment (axic) [inline test/index.js]: Same here.

File: test/index.js (hunk 2 above)
Comment (holgerd77) [reply test/index.js]: This has been done on a later commit, comment is outdated.

File: test/index.js (hunk 3 above)
Comment (holgerd77) [reply test/index.js]: Same here.

**Comment by holgerd77** [comment]
@axic Ok, did the renaming. This should be ready now. Will rebase/update the release commit once you approve/merge this.

**Comment by holgerd77** [comment]
@axic Have got 15 min left, if you do a speed-review, I could prepare the other PR! :-)

**Pull request merged**